x509/src/attr.rs

```diff
@@ -1,7 +1,11 @@
 //! Attribute-related definitions as defined in X.501 (and updated by RFC 5280).
 
+use alloc::vec::Vec;
+use core::fmt::Write;
```

Comment:
If you use:

```suggestion
use core::fmt::{self, Write};
```

...you can replace the instances of `core::fmt::*` with `fmt::*` below.

Reply:
I actually find that less clear.

Reply:
I think that's far and away the idiomatic thing to do.

Ballparking from GitHub search results:

- `impl fmt::`: [227,914 code results](https://github.com/search?l=Rust&q=%22impl+fmt%3A%3A%22&type=Code)
- `impl core::fmt::`: [7,588 code results](https://github.com/search?l=Rust&q=%22impl+core%3A%3Afmt%3A%3A%22&type=Code)

Reply:
Fixed.